docs/git-resolver.md

```diff
@@ -6,11 +6,13 @@ This Resolver responds to type `git`.
 
 ## Parameters
 
-| Param Name   | Description                                                                  | Example Value                                               |
-|--------------|------------------------------------------------------------------------------|-------------------------------------------------------------|
-| `url`        | URL of the repo to fetch.                                                    | `https://github.com/tektoncd/catalog.git`                   |
-| `revision`   | Git revision to checkout a file from. This can be commit SHA, branch or tag. | `aeb957601cf41c012be462827053a21a420befca` `main` `v0.38.2` |
-| `pathInRepo` | Where to find the file in the repo.                                          | `/task/golang-build/0.3/golang-build.yaml`                  |
+| Param Name   | Description                                                                                                       | Example Value                                               |
+|--------------|-------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `url`        | URL of the repo to fetch and clone anonymously. Either `url` or `org` and `repo` must be specified, but not both. | `https://github.com/tektoncd/catalog.git`                   |
+| `repo`       | The repository to find the resource in. Either `url` or `org` and `repo` must be specified, but not both.         | `pipeline`, `test-infra`                                    |
+| `org`        | The organization to find the repository in. Default can be set in [configuration](#configuration).                | `tektoncd`, `kubernetes`                                    |
```

Comment:
It's `url` or `repo`+`org` right ? Not sure how to make it clear in the parameter list but we should try 👼🏼

Reply:
Yeah, the wording could be improved, but I haven't come up with better yet.